src/harness/sas_testcase.py

```diff
@@ -141,3 +141,24 @@ def TriggerDailyActivitiesImmediatelyAndWaitUntilComplete(self):
     while not self._sas_admin.GetDailyActivitiesStatus()['completed']:
       time.sleep(10)
     signal.alarm(0)
+    
+  def TriggerFullActivityDumpAndWaitUntilComplete(self):
+    self._sas_admin.TriggerFullActivityDump()
```

Comment:
It's probably slightly better to reverse the order of these two lines to avoid a race condition.

Reply:
Please take another look at this.